packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -1,30 +1,35 @@
 /* eslint-disable import/no-cycle */
 import { Db } from '..';
-import { CredentialsService } from './credentials.service';
 import { CredentialsEntity } from '../databases/entities/CredentialsEntity';
 import { SharedCredentials } from '../databases/entities/SharedCredentials';
 import { User } from '../databases/entities/User';
+import { CredentialsService } from './credentials.service';
 import { RoleService } from '../role/role.service';
 
 export class EECredentialsService extends CredentialsService {
 	static async isOwned(
 		user: User,
 		credentialId: string,
 	): Promise<{ ownsCredential: boolean; credential?: CredentialsEntity }> {
-		const sharing = await this.getShared(user, credentialId, ['credentials'], {
+		const sharings = await this.getSharings(user, credentialId, ['credentials'], {
```

Comment:
For cred deletion and for this helper, we only need `findOne()` to check for access, but this would require duplicating `getSharings` or adding yet another flag to it, none of which are ideal. Thoughts?